content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -110,48 +63,87 @@ By default the `host_ip` is set to `127.0.0.1` i.e. localhost.
 host_ip: 127.0.0.1
 ```
 
-
 ---
 
 ### Select SODA Projects to install
-In the same `group_vars/common.yml` file, modify the `deploy_project` variable to select the projects to install. 
-Currently the `deploy_project` variable takes `all`, `hotpot` and `gelato` as values. By default it is set to `all`.  
-`all` installs SODA On Premise and SODA Multicloud (gelato).  
-`hotpot` installs SODA On Premise only.  
-`gelato` installs SODA Multicloud only.  
 
-```bash
-# This field indicates which project should be deploy
-# 'hotpot', 'gelato' or 'all'
-deploy_project: all  #all refers to hotpot + gelato
-```
-**_Note: Delfin and other member projects will be added shortly._**
+
+To install SODA Projects and enable the different features, variables have to be modified in the respective files as below:
+
+**_Note: Hotpot ,Gelato & Delfin can be used either through dashboard or REST APIs._**
 
 ---
 #### Enable Delfin installation
 delfin (Dolphin in Spanish!), the SODA Infrastructure Manager project is an open source project to provide unified, intelligent and scalable resource management, alert and performance monitoring. It covers the resource management of all the backends & other infrastructures under SODA deployment. It also provides alert management and metric data (performance/health) for monitoring and further analysis.
 
-To install Delfin, update the file `group_vars/delfin.yml` and change the value of `enable_delfin` to `true`
+  -Enable following configration  
+    - In file installer/ansible/group_vars/delfin.yml `enable_delfin: true`  
+    - In file installer/ansible/group_vars/srm-toolchain.yml `install_srm_toolchain: true`  
+    - In file installer/ansible/group_vars/dashboard.yml `enable_dashboard: true`  
+    - **[Click here](#configure-delfin-installation) for more config**
 
-```bash
-# Install delfin (true/false)
-enable_delfin: true
-```
+---
+
+
+#### Enable Hotpot  
+ Hotpot installs SODA On Premise only.  
+- Enable following configurations
+  - In file installer/ansible/hotpot.yml  update the value  `enable_hotpot : true`.
+  - In file installer/ansible/group_vars/dashboard.yml `enable_dashboard: true`
+  - In file installer/ansible/group_vars/common.yml `host_ip : <User's IP address, eg. 127.0.0.1>`  
+  - **[Click here](#configure-soda-on-premise-installation) for more config**  
+
+#### Enable Gelato
```

Comment:
Enable Strato.
let's use the project name Strato in the description. Mention the variable as gelato.